cmd/hyperfed/main.go

```diff
@@ -44,8 +42,6 @@ import (
 )
 
 func main() {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
this is fine cuz acc. to https://github.com/golang/go/issues/54880 global generator is seeded at startup i think. 
also i couldnt find any references as to why this is needed here but even if its used this change is fine i think